src/Widgets/PreferencesDialog.vala

```diff
@@ -119,6 +163,8 @@ public class ENotes.PreferencesDialog : Gtk.Dialog {
         var switch_box_ln = new Gtk.Box (Gtk.Orientation.HORIZONTAL, 0);
         switch_box_ln.add (line_numbers_switch);
 
+        sidebar_switch = new ENotes.PreferencesDialogSwitchBox ("Keep Sidebar Visible:", settings.keep_sidebar_visible);
```

Comment:
When using translations, the translated marks need to be on the string itself, not like on line 44 on PreferencesSwitchBox

https://elementary.io/docs/code/getting-started#Adding-Translations